### DISCUSSION TITLE
Decision model: Survival outcomrs

### DISCUSSION THREAD

**Discussion by mvb17** [MSKCC-Epi-Bio/decisioncurveanalysis#6] [General]
Hi what if for the example you gave in the survival outcomes section, for family history I wanted to plot a predictive model for people with 
Family history income = <51,000
Family history income = 51000-60000

*lets assume that family history was family history income

Using stata how would I modify the code you give ? 

Also can I plot the upper bounds and lower bounds in the decision model ?

**Comment by VickersA** [comment]
what is the issue here? that family income is categorical with more than two categories? you'd just use the "i." command i.e. 

xi: stcox i.family_income
